app/mender.go

```diff
@@ -179,15 +179,6 @@ func (m *Mender) IsAuthorized() bool {
 }
 
 func (m *Mender) Authorize() menderError {
```

Comment:
I'm not 100% sure you remember, but in my first PR we've had ForceAuthorize exactly for this reason... your implementation is way more robust, though, and it makes a lot of sense to always fetch a new JWT token when calling this function. Well done.

Reply:
Ok, it's possible I did not fully grasp the consequences at the time. I have also learned a lot about this as I have worked on it.